src/components/form/controls.tsx

```diff
@@ -57,6 +57,10 @@ export const Input = styled.input<ValidationProps>`
   border: 0;
   background: rgba(122, 143, 204, 0.2);
 
+  &:hover {
+    background: rgba(122, 143, 204, 0.5);
```

Comment:
you should use the available `polish` (it's a styled component companion) and the function `rgba` to use real values. The rgba values are not natural and you won't find them when you are going to replace/update color values in the future.